dev-docs/bidders/exadsadserver.md

```diff
@@ -0,0 +1,487 @@
+---
+layout: bidder
+title: EXADS
+description: EXADS Bidder Adapter
+pbjs: true
+pbs: false ?
+biddercode: exadsadserver
+gvl_id: 1084 ? // found here https://iabeurope.eu/vendor-list-tcf/
+tcfeu_supported: true?
+usp_supported: false?
+media_types: banner, video, native
+gpp_sids: ?
+schain_supported: ?
+dchain_supported: ?
+safeframes_ok: ?
+deals_supported: ?
+floors_supported: ?
+ortb_blocking_supported: ?
+---
+
+### Note
+
+Module that connects to EXADS’ bidder for bids.
+
+### Configuration
```

Comment:
@giuseppe-exads Before going into various configuration example, I would first add a simple and minimal example, something like you've done for the "RTB Banner 2.4". Then, I would add a section for each format with details of their specificity and configuration, from the simpler to the more complex one.

Reply:
@broucz In the beginning, before the configuration section, I've just added one very minimal snipped code without optional parameters and a brief description, in order to give a first idea about how to use the adapter.

After that, I've nearly maintained the configuration section as now because it is across over all ad formats. The only thing that I did is adding two sub paragraphs, generic and video settings, to make it more clear.

Anyway we can continue improving it but consider, as you know, that It is not the end user manual.